packages/probing-eservice-registry-updater/src/services/operationsService.ts

```diff
@@ -0,0 +1,33 @@
+import { SaveEserviceApi } from "../model/models.js";
+import { ZodiosInstance } from "@zodios/core";
+import { Api } from "pagopa-interop-probing-eservice-operations-client";
+import { logger } from "pagopa-interop-probing-commons";
+import {
+  apiSaveEserviceError,
+  makeApplicationError,
+} from "../model/domain/errors.js";
+
+export const operationsServiceBuilder = (
+  operationsApiClient: ZodiosInstance<Api>
+) => {
+  return {
+    async saveEservice({ params, payload }: SaveEserviceApi): Promise<void> {
+      try {
+        await operationsApiClient.saveEservice(payload, { params });
+
+        logger.info(
+          `Saving eService with eserviceId/versionId: ${params.eserviceId}/${params.versionId}`
+        );
```

Comment:
it's better log before doing the action, in case of failures
```suggestion
        logger.info(
          `Saving eService with eserviceId/versionId: ${params.eserviceId}/${params.versionId}`
        );

        await operationsApiClient.saveEservice(payload, { params });
```